1_The_Physical/Guess_Point_Level_1_Stateless/1_Red_Green_Refactor/1_2_Palindrome/src/index.spec.ts

```diff
@@ -1,7 +1,30 @@
-import { reverseAString } from './index';
+import { checkPalindrome, reverseAString } from './index';
 
 describe('palindrome checker', () => {
     it('reverses a string', () => {
         expect(reverseAString('bill')).toBe('llib');
     });
-})
\ No newline at end of file
+
+    it('returns false if it is not a palindrome', () => {
```

Comment:
It's better to avoid technical words in the spec name, for example we could name this test
```typescript
it('knows that "bill" is not a palindrome', () => ...)